addons/base_vat/models/res_partner.py

```diff
@@ -73,6 +73,7 @@
     'sk': 'SK2022749619',
     'sm': 'SM24165',
     'tr': _('TR1234567890 (VERGINO) or TR17291716060 (TCKIMLIKNO)'),  # Levent Karakas @ Eska Yazilim A.S.
+    'uy': _('\'NNNNNNNNNNNN\' (should be 12 digits)'),
```

Comment:
I will say that we tend to put a fake number that would be validated, as it's easier during the functional tests (when we need a VAT number on a partner) but it's fine

Reply:
I replace the  NNNNNNNNNNNN  with a generic rut number (one that use for testing - not real rut)